docs/tutorials/aws-sd.md

```diff
@@ -226,6 +226,16 @@ spec:
     ...
 ```
 
+## Enable sub-domain hosted zone match
```

Comment:
this doc file is about https://github.com/kubernetes-sigs/external-dns/tree/master/provider/awssd , I think you want to document https://github.com/kubernetes-sigs/external-dns/tree/master/provider/aws right?

Reply:
Right, I'll replace to /provider/aws.